src/others/ime/lv_ime_pinyin.h

```diff
@@ -25,6 +25,8 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
+const lv_obj_class_t lv_ime_pinyin_class;
```

Comment:
I think this needs to be `extern` and should be in the "GLOBAL VARIABLES" section.

Reply:
Done on 3d78b38b3